pyigm/abssys/utils.py

```diff
@@ -23,6 +23,7 @@
 from linetools.analysis import absline as ltaa
 from linetools.isgm.abscomponent import AbsComponent
 
+
 def chk_components(components, chk_match=False, chk_A_none=False, toler=0.2*u.arcsec):
     """ Performs checks on a list of components
```

Comment:
You are right.  I had copied over the whole file to get some methods that
should be in pyigm but forgot to strip out the component ones.
Fixed.